traveltimepy/dto/common.py

```diff
@@ -97,22 +199,51 @@ class Rectangle(BaseModel):
     max_lng: float
 
     def to_str(self):
-        return f'{self.min_lat},{self.min_lng},{self.max_lat},{self.max_lng}'
+        return f"{self.min_lat},{self.min_lng},{self.max_lat},{self.max_lng}"
+
+    def __hash__(self):
+        return hash((self.min_lat, self.max_lat, self.min_lng, self.max_lng))
+
+    def __eq__(self, other):
+        if not isinstance(other, Rectangle):
+            return NotImplemented
+        return (self.min_lat, self.max_lat, self.min_lng, self.max_lng) == (
+            other.min_lat,
+            other.max_lat,
+            other.min_lng,
+            other.max_lng,
+        )
 
 
 class Property(str, Enum):
-    TRAVEL_TIME = 'travel_time'
-    DISTANCE = 'distance'
-    ROUTE = 'route'
-    FARES = 'fares'
+    TRAVEL_TIME = "travel_time"
```

Comment:
Sure, I rolled that back.
It happened because I used `black` for formatting which (for good reasons IMHO) suggests double quotes, but that's probably a general discussion to have if `black`'s formatting style (which seems to be adopted by more and more projects) should be used here as well.